src/color/p5.Color.js

```diff
@@ -52,8 +52,18 @@ p5.Color = function(renderer, vals) {
 };
 
 /**
+ * This function returns the color formatted as a string. This can be useful
+ * for debugging, or for using p5.js with other libraries.
  * @method toString
- * @return {String}
+ * @param {Number|String} [format] How the color string will be formatted.
+ * Leaving this empty formats the string as rgba(r, g, b, a).
+ * '#rgb' '#rgba' '#rrggbb' and '#rrggbbaa' format as hexadecimal color codes.
+ * 'hex3' 'hex4' 'hex6' and 'hex8' return the same, without the '#''
```

Comment:
is this one necessary? it's adding in new language that we haven't really used elsewhere in the library. the other formats mostly align with inputs for the color functions and feel more intuitive. couldn't the user just use substring to remove the '#' if desired?

Reply:
I get where you're coming from, I was torn between trying to be easy to understand and trying to cover all the options someone might guess. Do you think it'd be better to remove the 'hex' options altogether, just have '#rrggbb' etc for the different hex color codes?